recipes/denoptim/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -euo
+
+# Build
+cd "$SRC_DIR"
+mkdir -p "$PREFIX/lib" "$PREFIX/bin"
+
+mvn clean package 
+cp "$SRC_DIR/target/denoptim-$PKG_VERSION-jar-with-dependencies.jar" "$PREFIX/lib"
+
+echo '#!/bin/bash' > "$PREFIX/bin/denoptim"
+echo 'java -jar "'$PREFIX'/lib/denoptim-'$PKG_VERSION'-jar-with-dependencies.jar" "$@"' >> "$PREFIX/bin/denoptim"
+
+chmod +x "${PREFIX}/bin/denoptim"
```

Comment:
The recipe is shipping a file in `/var/cache/fontconfig/`. Is this needed or just some install garbage?

Reply:
Uhm... I've tried on an ubuntu 20.04, but I don't get anything on /var/cache/fontconfig/. Whatever it is that is put there, it is not meant to be there and not needed.
Nevertheless, I've updated the build script to get rid of an error I got when running interactively on Ubuntu. Perhaps, the two things are somehow connected (I doubt...).

Reply:
It's still there. I suggest manually removing `PREFIX/var/cache` as the last build step.

Reply:
Done in 1850e8fb7e8abd9b0a9ed82cef1245e59b4e518f